waspc/data/Generator/templates/react-app/src/auth/user.ts

```diff
@@ -0,0 +1,27 @@
+// Since we can't deduplicate these helper functions in the server and the client
```

Comment:
I've already commented on that option here: https://github.com/wasp-lang/wasp/pull/1583#discussion_r1431424497 

As I said, I've considered it and went with a simpler solution of duplication. This can be the temporary solution if it's going to change with the restructuring.

Reply:
I commented again because you said in the comment in the code (the one I commented on here) that we can't deduplicate this, which is not correct, we can, but you decided we rather shouldn't as it is not worth it. So maybe let's update comment with that info?